python/ray/data/context.py

```diff
@@ -93,6 +94,15 @@
     "AWS Error UNKNOWN (HTTP status 503)",
 )
 
+DEFAULT_RETRIED_FILESYSTEM_ERRORS = (
+    "AWS Error INTERNAL_FAILURE",
+    "AWS Error NETWORK_CONNECTION",
+    "AWS Error SLOW_DOWN",
+    "AWS Error UNKNOWN (HTTP status 503)",
+    "AWS Error ACCESS_DENIED",
+    "AWS Error NETWORK_CONNECTION",
+)
```

Comment:
Consolidating all of the transient errors in one place so users don't need to configure retries for opening files/reading files/writing files separately.